src/components/common/NameRequestInfo.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <div id="name-request-summary">
+    <v-row id="name-request-info">
+      <v-col>
+        <label>
+          <strong>Name Request</strong>
+        </label>
+      </v-col>
+      <v-col>
+        <ul>
+          <li class="name-request-title mb-4">
+            <strong>{{ getBusinessIdentifier }}</strong> - {{ getNameRequestDetails.approvedName}}
+          </li>
+          <li>Entity Type: {{ entityTypeDescription() }}</li>
+          <li class="mb-4"> Request Type: {{ requestType() }}</li>
```

Comment:
FWIW, this is more of a personal preference thing (so I think it's OK as long as you're consistent -- and you are here), but I prefer to add top margins/padding to the lower element (since it "knows" what it comes after) instead of adding bottom margins/padding to the upper element (since it can only "assume" what come after it. This makes more sense when there's a lot of conditionals.

Reply:
Makes a lot of sense thanks! Updated.